From 1f2c6ed0b6a32182268cecb6db13efa504dcac17 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 10 Jan 2023 15:30:39 +0200
Subject: [PATCH] [SW-0] gaudi2: arm: ppboot: add 4k to sram section

It appears the Gaudi2 ARM .sram section in ppboot is full and CI fails
the build.
We do not use ARM ppboot on Gaudi2, but it is has not been removed from
builds.
Meanwhile to let CI pass, raise the PPBOOT_SIZE to 24KB.

Change-Id: Ie0093d346301f5d42148a420fc5903f19a0d5f21
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/arch/gaudi2/arm/common.h b/arch/gaudi2/arm/common.h
index 56a0437..dab3257 100644
--- a/arch/gaudi2/arm/common.h
+++ b/arch/gaudi2/arm/common.h
@@ -32,7 +32,7 @@
 #define HL_FW_BOOT_PTRS_BASE		0x60000
 #define HL_SPI_LEGACY_PREBOOT_OFF	0xC0000
 
-#define PPBOOT_SIZE			0x5000		/* 20KB */
+#define PPBOOT_SIZE			0x6000		/* 24KB */
 #define PREBOOT_SIZE			0x20000		/* 128KB */
 #define PREBOOT_TEXT_BASE		(PPBOOT_TEXT_BASE + PPBOOT_SIZE)
 
